lib/rucio/rse/protocols/gsiftp.py

```diff
@@ -14,6 +14,8 @@
  - Mario Lassnig, <mario.lassnig@cern.ch>, 2017
```

Comment:
Add author!

Reply:
you can even use the script:  ./tools/add_header {file} which will update the author list and put the right header